src/rime_api.h

```diff
@@ -467,7 +467,10 @@ typedef struct rime_api_t {
   Bool (*config_begin_map)(RimeConfigIterator* iterator, RimeConfig* config, const char* key);
   Bool (*config_next)(RimeConfigIterator* iterator);
   void (*config_end)(RimeConfigIterator* iterator);
-
+  
+  // Utilities
+  void (*set_page_size)(RimeSessionId session_id, int page_size);
```

Comment:
New API functions must be appended to the end of the struct, for new versions of rime dynamic library to be replaceable without having to recompile client executables.